ProjFS.Mac/PrjFSLib/PrjFSKextLogDaemon/PrjFSKextLogDaemon.cpp

```diff
@@ -0,0 +1,168 @@
+#include "../../PrjFSKext/public/PrjFSLogClientShared.h"
+#include "../PrjFSUser.hpp"
+#include <iostream>
+#include <OS/log.h>
+#include <IOKit/IOKitLib.h>
+#include <signal.h>
+
+struct LogConnectionState
```

Comment:
Why does this struct only have one member? We have a same named struct inside in prjfs-log.cpp but with different members. Can we rename this one or reuse that one?
```
struct LogConnectionState
{
    DataQueueResources dataQueue;
    unsigned lineCount;
};
```